File: leoz-ui/src/main/java/org/deku/leoz/bridge/MediaType.java
package org.deku.leoz.bridge;

/**
 * Created by masc on 27.09.14.
 */
public class MediaType {
    /**
     * JSON UTF-8 media/mime type.
     * Required for compatibility with WCF clients/hosts
     */
    public static final String APPLICATION_JSON_UTF8 = javax.ws.rs.core.MediaType.APPLICATION_JSON + ";charset=utf-8";
}

File: leoz-ui/src/main/java/org/deku/leoz/ui/bridge/MediaType.kt
package org.deku.leoz.ui.bridge

/**
 * Created by masc on 27.09.14.
 */
object MediaType {
    /**
     * JSON UTF-8 media/mime type.
     * Required for compatibility with WCF clients/hosts
     */
    const val APPLICATION_JSON_UTF8 = javax.ws.rs.core.MediaType.APPLICATION_JSON + ";charset=utf-8"
}
